draft-ietf-radext-radiusdtls-bis.md

```diff
@@ -381,6 +381,19 @@ The above requirements are a logical extension of the common practice where a cl
 
 In an ideal world, a proxy could also apply the suggestion of the previous section, by discarding Acct-Delay-Time from Accounting-Request packets, and replacing it with Event-Timestamp.  However, this process is fragile and is not known to succeed in the general case.
 
+## Session limits and timeout
+
+While RADIUS/UDP could be implemented mostly stateless (except for the requests in flight), both TCP/TLS as well as DTLS require state tracking of the underlying TLS connection and are thus subject to potential resource exhaustion. This is aggravated by the fact that radius client/servers often from long-running peer relationships translating to long-running connections.
```

Comment:
I don't understand this last sentence (and unfortunately I can't infer from context what word is missing)

Reply:
Ah, mabye its just the type r/from/form/, but the point was to emphasize that current setups often use long-lived TLS connections.

I will try to reword...

Reply:
ok, I think the from/form correction clarifies what is meant.
Let's see what the WG says